django/db/models/base.py

```diff
@@ -1926,6 +1926,9 @@ def _check_constraints(cls, databases):
                         id='models.W038',
                     )
                 )
+            for constraint in cls._meta.constraints:
+                if isinstance(constraint, UniqueConstraint):
+                    errors.extend(cls._check_local_fields(constraint.fields, "unique_constraint"))
```

Comment:
Error message doesn't contain information about specific constraint so we can merge this to a single message.